test/ads/vast/client/Companion.spec.js

```diff
@@ -0,0 +1,65 @@
+describe("Companion", function(){
+  it("must return an instance of Companion", function(){
+    var companion = Companion(xml.toJXONTree('<Companion><StaticResource></StaticResource></Companion>'));
+    assert.instanceOf(companion, Companion);
+  });
+
+  //Required Elements
+  describe("staticResource", function(){
+    it("must contain the static resource and mime type", function(){
+      var companionXML = '<Companion><StaticResource creativeType="image/jpeg"><![CDATA[http://www.example.com/image.jpg]]></StaticResource></Companion>';
+      var companion = Companion(xml.toJXONTree(companionXML));
+      assert.equal(companion.creativeResource, "http://www.example.com/image.jpg");
+      assert.equal(companion.creativeType, "image/jpeg");
+    });
+  });
+
+  //Optional Elements
+  describe("trackingEvents", function(){
+    /*jshint maxlen: 500 */
+    it("must be filled with all the tracking events that the xml provides", function(){
+      var companionXML = '<Companion>' +
```

Comment:
@massa84 Definitely, I just used the inline XML example from `Linear.js` as a guide to build the test and used the same tracking URL. Where would you want the external XML file to be?